index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Home Page</title>
+</head>
+<body>
+  <div class="grid-container">
+    <nav class="navbar">
+      <div class="hamburger small-screen">
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+      </div>  
+      <a href="#" class="brand"><img src="images/logo.png" alt="logo"  class="brand">Nyegenyege</a>
+      <ul class="nav-list flex-row">
+        <li class="nav-item "><a href="#" class="active">About</a></li>
+        <li class="nav-item"><a href="#">program</a></li>
+        <li class="nav-item"><a href="#">Join</a></li>
+        <li class="nav-item"><a href="#">Sponsor</a></li>
```

Comment:
It was not possible to navigate between the pages. I couldn't navigate from the `Home` page to the `About` page. Also, your `mobile hamburger menu` is not responding to click so I am unable to check what the menu likes. Kindly fix it.